consensus/src/simplex/prover.rs

```diff
@@ -130,7 +130,7 @@ impl<C: Scheme, H: Hasher> Prover<C, H> {
         namespace: &[u8],
     ) -> Option<(View, View, Digest, Vec<PublicKey>)> {
         // Ensure proof prefix is big enough
-        let digest_len = H::len();
+        let digest_len = H::DIGEST_LENGTH;
```

Comment:
Because this is a const, I think we can call inline rather than renaming it as a new variable.